test/NuGet.Core.Tests/NuGet.Versioning.Test/VersionRangeFloatParsingTests.cs

```diff
@@ -164,8 +186,16 @@ public void VersionRangeFloatParsing_FloatPrereleasePrefixSemVerLabelMix()
         [InlineData("1.*.0-beta-*")]
         [InlineData("1.*.0-beta")]
         [InlineData("1.0.0.0.*")]
-        //[InlineData("1.0.0*")]
         [InlineData("=1.0.*")]
+        [InlineData("1.0.0+*")]
+        [InlineData("1.0.**")]
+        [InlineData("1.0.*-*bla")]
+        [InlineData("1.0.*-*bla+*")]
+        [InlineData("**")]
+        [InlineData("1.0.0-preview.*+blabla")]
+        [InlineData("1.0.*--")]
+        [InlineData("1.0.*-alpha*+")]
+        [InlineData("1.0.*-")]
```

Comment:
What about an Empty string? Perhaps we should test that whitespace, random character, integer, etc are considered invalid rather than an unhandled exception?

Reply:
Empty and white space strings were not handled by the code, but that was always the case. 
I have added an extra change to handle that more gracefully. 

The reality is that floating range parse is never invoked in production code and the VersionRange.Parse method has a try catch for this.